examples/kafka/cluster.yaml

```diff
@@ -0,0 +1,23 @@
+# It can take approximately 15-25 minutes to create a cluster.
+apiVersion: kafka.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: kafka-test-server
+spec:
+  forProvider:
+    brokerNodeGroupInfo:
+      clientSubnets:
+        - subnet-f2ccb898
+        - subnet-aac361d6
+      instanceType: kafka.t3.small
+      storageInfo: 
+        ebsStorageInfo:
+          volumeSize: 1
+    clusterName: kafka-test-server
```

Comment:
Is this the unique identifier of the MSK cluster? Should it be the external name?

https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#external-name

Reply:
Answering my own question - the ARN is the unique identifier, so no this should not be the external name. :)

Reply:
yes ;)